src/reports/reportbase.cpp

```diff
@@ -35,7 +35,7 @@ wxString mmPrintableBase::local_title() const
     if (!m_date_range) 
         return m_local_title; 
     else 
-        return m_local_title + _(" - ") + m_date_range->local_title();
+        return m_local_title + " - " + m_date_range->local_title();
```

Comment:
It should be like (not sure):
~~~~~
return wxString::Format(_("%s1 - %s2"), m_local_title,m_date_range->local_title());
~~~~~